apps/web/views.py

```diff
@@ -37,12 +37,16 @@
 from lib.departments import get_department_name
 from lib import constants
 
+import logging
+
 import uuid
 from google.cloud import pubsub_v1
 
+logger = logging.getLogger('layuplist.web')
+
 pub_sub_publisher = pubsub_v1.PublisherClient()
 topic_paths = {
-    'course-views': pub_sub_publisher.topic_path(os.environ['GCLOUD_PROJECT_ID'], 'course-views')
+    'course-views': pub_sub_publisher.topic_path('d-planner-data', 'course-views')
```

Comment:
add note in PR desc about this? seems not debugging related

Reply:
This was a "I'll fix it later thing lmao" but basically during initial migration `os.environ` couldn't read the `.env` so I just hardcoded it to get over that hump.